TodoAppWithSpringboot/src/main/java/com/webonise/service/UserService.java

```diff
@@ -0,0 +1,46 @@
+package com.webonise.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.webonise.dao.UserDao;
+import com.webonise.model.User;
+import lombok.extern.slf4j.Slf4j;
+
+@Service
+@Slf4j
+public class UserService {
+	
+	@Autowired
+	private UserDao userDao;
+	
+	public User findByUsername(String username) throws Exception {
+		try {
+			return userDao.findByUsername(username);
+		} catch (Exception ex) {
+			log.info("Exception occured during retrival of user by it's username.");
+			throw new Exception("Find by username was unable to execute." , ex);
+		}
+	}
+	
+	public User getMaxId() throws Exception {
+		try {
+			return new User((int)userDao.count() + 1);
+		} catch (Exception ex) {
```

Comment:
We should not handle exception just we have handled, It should be proper. Avoid throwing Generalized Exceptions should be more specialized or customized.